src/controllers/dashboard/scheduledtasks/scheduledtasks.js

```diff
@@ -66,7 +66,8 @@ define(["jQuery", "loading", "events", "globalize", "serverNotifications", "huma
         var html = "";
         if (task.State === "Idle") {
             if (task.LastExecutionResult) {
-                html += globalize.translate("LabelScheduledTaskLastRan").replace("{0}", humaneDate(task.LastExecutionResult.EndTimeUtc)).replace("{1}", humaneElapsed(task.LastExecutionResult.StartTimeUtc, task.LastExecutionResult.EndTimeUtc));
+                html += globalize.translate("LabelScheduledTaskLastRan", datefns.formatDistanceToNow(Date.parse(task.LastExecutionResult.EndTimeUtc), { addSuffix: true, locale: dfnshelper.getLocale() }),
+                    datefns.formatDistance(Date.parse(task.LastExecutionResult.StartTimeUtc), Date.parse(task.LastExecutionResult.EndTimeUtc), { addSuffix: false, locale: dfnshelper.getLocale() }));
```

Comment:
```suggestion
                const lastExecutionResultToNow = datefns.formatDistanceToNow(Date.parse(task.LastExecutionResult.EndTimeUtc), { addSuffix: true, locale: dfnshelper.getLocale() })
                const lastExecutionResultInterval = datefns.formatDistance(Date.parse(task.LastExecutionResult.StartTimeUtc), Date.parse(task.LastExecutionResult.EndTimeUtc), { addSuffix: false, locale: dfnshelper.getLocale() });
                html += globalize.translate("LabelScheduledTaskLastRan", lastExecutionResultToNow, lastExecutionResultInterval);
```

This is a bit long and complex, so feel free to separate it into variables.

You could also probably move the following to the helper module, as it seems to be quite common:
```
{ addSuffix: true, locale: dfnshelper.getLocale() }
```
Export it as something like:
```javascript
export const localeWithSuffix = { addSuffix: true, locale: dfnshelper.getLocale() }
```